test/test.h

```diff
@@ -84,7 +84,7 @@ using LiveTrafficCustomize = std::function<void(valhalla::baldr::GraphReader&,
 void customize_live_traffic_data(const boost::property_tree::ptree& config,
                                  const LiveTrafficCustomize& setter_cb);
 
-using HistoricalTrafficCustomize = std::function<std::vector<int16_t>(DirectedEdge&)>;
+using HistoricalTrafficCustomize = std::function<std::array<float, kBucketsPerWeek>(DirectedEdge&)>;
```

Comment:
this is way more useful if you give it the raw speeds and let it convert it to the format stored in the tiles